check_missing_installations.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
```

Comment:
How about adding author and license in the header section?

Reply:
sure